etl-api/tests/integration/pipelines_test.rs

```diff
@@ -147,18 +147,43 @@ async fn setup_pipeline_with_source_db() -> (TestApp, String, i64, sqlx::PgPool,
         new_pipeline_config(),
     )
     .await;
+
+    // We run the migrations to create all the tables used by `etl`.
+    run_etl_migrations_on_source_db(&source_pool).await;
+
     (app, tenant_id, pipeline_id, source_pool, source_db_config)
 }
 
+/// Runs etl migrations on the source database.
+async fn run_etl_migrations_on_source_db(source_pool: &PgPool) {
+    // Create the `etl` schema first.
+    sqlx::query("create schema if not exists etl")
+        .execute(source_pool)
+        .await
+        .unwrap();
+
+    // Set the `etl` schema as search path (this is done to have the migrations metadata table created
+    // by sqlx within the `etl` schema).
+    sqlx::query("set search_path = 'etl';")
+        .execute(source_pool)
+        .await
+        .unwrap();
+
+    // Run replicator migrations to create the state store tables.
+    sqlx::migrate!("../etl-replicator/migrations")
+        .run(source_pool)
+        .await
+        .unwrap();
+}
+
```

Comment:
The `set search_path = 'etl'` might not work if the connection it picks from the pool is different than the connection used to run subsequent queries since this will stay in effect on only on the connection in which it was run.